PDF_UA/WCAG-21-Complete.xml

```diff
@@ -6,7 +6,7 @@
     </details>
     <hash></hash>
     <rules>
-        <rule object="SARepeatedCharacters">
+        <rule object="SARepeatedCharacters" tags="minor,syntax,text">
```

Comment:
I don't think "syntax" is appropriate here. This is the rule we added as an optional requirement included into various guides on accessibility. I'd suggest reserving "syntax" to issues in low level (COS, PD) PDF syntax